package.json

```diff
@@ -1,54 +1,43 @@
 {
-  "name": "kumascript",
-  "description": "Scripting for MDN",
-  "version": "0.1.0",
-  "license": "MPL-2.0",
-  "homepage": "https://github.com/mdn/kumascript",
-  "bugs": "https://github.com/mdn/kumascript/issues",
-  "author": "Les Orchard <lorchard@mozilla.com> (http://lmorchard.com/)",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/mdn/kumascript.git"
-  },
-  "main": "./lib/kumascript/index",
-  "scripts": {
-    "start": "make run",
-    "test": "make test"
-  },
-  "engines": {
-    "node": "10.14.2"
-  },
-  "directories": {
-    "lib": "./lib"
-  },
-  "dependencies": {
-    "async": "2.1.4",
-    "ejs": "2.5.7",
-    "express": "4.14.0",
-    "fibers": "3.1.1",
-    "hirelings": "0.4.3",
-    "mdn-browser-compat-data": "0.x",
-    "memcached": "2.2.2",
-    "morgan": "1.7.0",
-    "nconf": "0.8.4",
-    "newrelic": "4.1.0",
-    "node-statsd": "0.1.1",
-    "pegjs": "0.7.0",
-    "request": "2.79.0",
-    "underscore": "1.8.3",
-    "winston": "2.3.0"
-  },
-  "devDependencies": {
-    "mocha": "3.4.1",
-    "chai": "3.5.0",
-    "chai-as-promised": "7.0.0",
-    "sinon": "2.3.1",
-    "mocha-junit-reporter": "1.13.0",
-    "jshint": "2.9.4",
-    "ejs-lint": "0.3.0",
-    "jsonlint-cli": "1.0.1",
-    "tmp": "0.0.31",
-    "jsdom": "11.3.0",
-    "extend": "3.0.1"
-  }
+    "name": "kumascript",
+    "description": "Scripting for MDN",
+    "version": "0.2.0",
+    "license": "MPL-2.0",
+    "homepage": "https://github.com/mdn/kumascript",
+    "bugs": "https://github.com/mdn/kumascript/issues",
+    "author": "Les Orchard <lorchard@mozilla.com> (http://lmorchard.com/) and David Flanagan <dflanagan@mozilla.com>",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/mdn/kumascript.git"
+    },
+    "main": "./index",
+    "scripts": {
+        "start": "node run.js",
+        "test": "jest",
+        "lint": "eslint *.js src tests",
+        "lint-json": "find . -name '*.json' -not -path './node_modules/*' -print0 | xargs -t -n1 -0 jsonlint -q",
+        "format": "./node_modules/.bin/prettier --write --require-pragma *.js src/*.js tests/*.js tests/macros/*.js"
+    },
+    "engines": {
+        "node": "10.14"
+    },
+    "dependencies": {
+        "ejs": "2.6.1",
+        "express": "4.16",
+        "lru-cache": "5.1.1",
+        "mdn-browser-compat-data": "0.0.59",
```

Comment:
For now at least, I think we should keep this at:
```
"mdn-browser-compat-data": "0.x",
```
We did that originally so we could easily add the latest version of `mdn-browser-compat-data` without submitting a Kumascript PR (which happens almost every week on Thursday), and we have a [line in the Kumascript Dockerfile](https://github.com/mozilla/kuma/blob/9589b848ff098cce4c4827fc15668e4bae4d3c5a/docker/images/kumascript/Dockerfile#L29) explicitly for taking advantage of that and updating to the latest.

Reply:
Thank you for noticing that! I pinned it to that specific version because that was what was in use when I generated my test corpus with the old KumaScript, and in order to compare compat tables, I needed the same version. But that was just supposed to be temporary!

Reply:
Fixed in 5b40b6e